consumer/guides/mraa/install.md

```diff
@@ -2,12 +2,22 @@
 title: Guides
 permalink: /documentation/consumer/guides/mraa/install.md.html
 redirect_from: /documentation/ConsumerEdition/guides/mraa/install.md.html
+
 ---
 
 # Installation guide for MRAA library on 96Boards CE
 
 Following instructions provides information about installing the MRAA library as a package and building from source.
 
+## MRAA Overview
+**[MRAA](https://iotdk.intel.com/docs/master/mraa/)** (libmraa) is a C/C++ low level skeleton library for communication on GNU/Linux platforms.
+It binds to Python, Javascript and Java to interface with the I/O on ARM platforms such as the DragonBoard 820c.
```

Comment:
Please don't specify any particular board name here. This guide is meant for all of the 96Boards